packages/suite-web/e2e/tests/onboarding/t1b1-recovery-basic.test.ts

```diff
@@ -3,13 +3,13 @@
 
 // todo: this started to fail mysteriously after merging new base image. Skipping it for now and will investigate.
 describe.skip('Onboarding - recover wallet T1B1', () => {
-    before(() => {
+    beforeEach(() => {
         cy.task('startEmu', { model: 'T1B1', version: '1-latest', wipe: true });
         cy.task('startBridge');
 
         cy.viewport(1440, 2560).resetDb();
         cy.prefixedVisit('/');
-        cy.task('startEmu', { wipe: true });
+        cy.disableFirmwareHashCheck();
```

Comment:
the test is skipped, I tried it locally and it's still broken :x:
Fixing it is out of the scope, but I at least verified that `disableFirmwareHashCheck` is necessary, and that it works (proceed to next parts where the test fails). So when someone wants to fix the test in future, they'll have one less thing to fix :slightly_smiling_face: 

Though I fixed two more problems:
- idk why, but `before` does not work - it fails to initialize emu, while `beforeEach` works :heavy_check_mark: 
- the second invocation of `startEmu` was killing the T1B1 which this test wants, and loading T2T1 instead :x: